pages/linux/tcptraceroute.md

```diff
@@ -0,0 +1,28 @@
+# tcptraceroute
+
+> A traceroute implementation using TCP packets.
+> More information: <https://github.com/mct/tcptraceroute>.
+
+- Trace the route to a host:
+
+`tcptraceroute {{host}}`
+
+- Specify destination port and packet length in bytes:
+
+`tcptraceroute {{host}} {{destination_port}} {{packet_length}}`
+
+- Specify local source port and source address:
+
+`tcptraceroute {{host}} -p {{source_port}} -s {{source_address}}`
+
+- Set initial and maximum TTL:
+
+`tcptraceroute {{host}} -f {{first_ttl} -m {{max_ttl}}`
+
+- Specify wait time and number of probes for each hop:
+
+`tcptraceroute {{host}} -w {{wait_time}} -q {{number_of_queries}}`
+
+- Specify interface with HTTP (port 80) as the destination port:
```

Comment:
You could always type port x instead, where x is a port number, or you can type http, https, ftp , ssh... . Though I'm not sure if a simple name can be used for every single port.

I think removing it might be simpler --> `tcptraceroute {{host}} -i {{interface}}`

What do you think?

Reply:
That would probably be a good idea. Since we've specified the port number in a different example, we probably don't need to include it in this one too.